workflow/scripts/preprocess_mutational_variants.py

```diff
@@ -0,0 +1,31 @@
+import sys
```

Comment:
Is this script used anywhere? I can't find a rule that refers to it.

Reply:
Good catch. This was a leftover from the previous try of implementing SigProfilerAssignment.